org-service/test/controllers/BaseControllerTest.java

```diff
@@ -0,0 +1,33 @@
+package controllers;
```

Comment:
Can this be a separate module called "test-harness" and put these controllers, helper classes there? I see this is probably duplicate code from user-service. Feel free to make this common to both these services.

Reply:
This code will be changed and will be user / org specific , right now we don't have any use cases so this looks same but, it will contains methodswhich will return user/org specific data for test cases.